lib/tasks/check-links/link_checker.rake

```diff
@@ -1,31 +1,20 @@
 require 'local-links-manager/distributed_lock'
-require 'local-links-manager/check_links/homepage_status_updater'
-require 'local-links-manager/check_links/link_status_updater'
+require 'local-links-manager/check_links/link_status_requester'
 
 desc "Check links"
 task "check-links": :environment do
   service_desc = "Local Links Manager link checker rake task"
-  LocalLinksManager::DistributedLock.new('check-links').lock(
-    lock_obtained: ->() {
-      begin
-        Rails.logger.info("Lock obtained, starting link checker")
-        Services.icinga_check(service_desc, true, "Lock obtained, starting link checker")
+  begin
```

Comment:
Hmm, perhaps I misunderstood what the point of the lock was, I figured it was there to avoid someone accidentally running the take command twice. Since the rake task now only takes a few minutes (it sends off requests to the `link-checker-api` which then responds through a web hook) is it necessary to have the lock?